docs/samples-and-tutorials/index.md

```diff
@@ -1,111 +1,122 @@
 
-## General
+## .NET Core
 
 ### Samples
 
 **[Unit Testing in .NET Core using dotnet test](../core/testing/unit-testing-with-dotnet-test.md)**
 
-This guide shows how to create an ASP.NET Core web application and the associated unit tests. It will start by creating a simple web service application and then add tests, and continue by creating more tests to guide implementing new features. The completed code is available [in the core-docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/core/getting-started/unit-testing-using-dotnet-test).
+This guide shows you how to create an ASP.NET Core web app and associated unit tests. It starts by creating a simple web service app and then adds tests. It continues with creating more tests to guide implementing new features. The [completed sample](https://github.com/dotnet/docs/tree/master/samples/core/getting-started/unit-testing-using-dotnet-test) is available in the dotnet/docs repository on GitHub.
 
 ### Tutorials
 
 **[Writing .NET Core console apps using the CLI tools: A step-by-step guide](../core/tutorials/using-with-xplat-cli.md)**
 
-This guide will show you how to use the .NET Core CLI tooling to build cross-platform console apps.  It will start with the most basic console app and eventually span multiple projects, including testing. You'll add these features step-by-step, building on what you've already seen and built. A finished version of the code you'll build is located [in the core-docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/core/console-apps).
+This guide shows you how to use the .NET Core CLI tooling to build cross-platform console apps. It starts with a basic console app and eventually spans multiple projects, including testing. You add features step-by-step, building your knowledge as you go. The [completed sample](https://github.com/dotnet/docs/tree/master/samples/core/console-apps) is available in the dotnet/docs repository on GitHub.
 
 **[Writing Libraries with Cross Platform Tools](../core/tutorials/libraries.md)**
 
-This sample covers how you can write libraries for .NET using cross-platform CLI tools.  They provide an efficient and low-level experience that works across any supported OS.
-The completed code is available [in the core-docs repository on GitHub](https://github.com/dotnet/docs/tree/master/samples/framework/libraries/frameworks-library).
+This sample covers how to write libraries for .NET using cross-platform CLI tools. These tools provide an efficient and low-level experience that works across any supported operating system. The [completed sample](https://github.com/dotnet/docs/tree/master/samples/framework/libraries/frameworks-library) is available in the dotnet/docs repository on GitHub.
 
-## C# Language
+## ASP.NET Core
+
+The [ASP.NET Core tutorials](https://docs.microsoft.com/aspnet/core/tutorials/) include step-by-step guides and sample code for:
```

Comment:
Samples are in other docs also.  You could mention that many of the ASP.NET Core articles that aren't tutorials also have samples associated with them